modules/bichard7_ecs_task/data.tf

```diff
@@ -26,7 +26,7 @@ data "template_file" "bichard_fargate" {
     HEALTH_CHECK_TAC      = var.health_check_tac
     DEPLOY_ENV            = local.deploy_env
     TAC_SUFFIX            = local.tac_suffix
-    DISABLE_MDB           = var.service_type == "web" ? "true" : ""
+    DISABLE_MDB           = var.service_type == "backend" ? "" : "true"
```

Comment:
because now we have two services that need this enabled, we explicitly set the var when it is backend